src/framework.cpp

```diff
@@ -41,11 +43,18 @@
 #define CONSTRAINTS_SIMPLIFY_THRESHOLD 10000U
 #define MAX_FARKAS_CST 2000
 
+// This value represents the upper bound on w in case the dependence is short.
+#define SHORT_DEP_DISTANCE 10
+
 static int pluto_dep_satisfies_instance(const Dep *dep, const PlutoProg *prog,
                                         unsigned level);
 static int pluto_dep_remove_satisfied_instances(Dep *dep, PlutoProg *prog,
                                                 unsigned level);
 
+static bool is_stmt_in_stmt_list(int stmt_id, std::vector<Stmt *> stmts);
```

Comment:
Pass read-only std::vector arguments by const reference! Leads to unnecessary copies otherwise.